composer.json

```diff
@@ -27,11 +27,15 @@
   "require-dev": {
     "squizlabs/php_codesniffer": "^2.0",
     "fzaninotto/faker": "^1.7",
-    "phpunit/phpunit": "^4.0"
+    "phpunit/phpunit": "^4.8.36"
```

Comment:
The `PHPUnit\Framework\TestCase` class has supported since the `phpunit-4.8.36` version.